gatsby-node.js

```diff
@@ -100,12 +100,6 @@ exports.onPostBuild = async function (
 
     const objects = await transformer(result);
```

Comment:
```suggestion
    const objects = await transformer(result).map(object => ({ objectID: object.objectID || object.id, ...object }));
```

this is very similar to your fix